lib/rubocop/cop/lint/empty_conditional_body.rb

```diff
@@ -96,7 +96,8 @@ def remove_empty_branch(corrector, node)
         end
 
         def correct_other_branches(corrector, node)
-          return unless (node.if? || node.unless?) && node.else_branch
+          return unless correctable_condition?(node) && node.else_branch
```

Comment:
Can you use `node.if_type?` instead of the new `correctable_condition?` method?

```suggestion
          return unless node.if_type? && node.else_branch
```

Reply:
That looks better. Thank you!